File: Python Course/Software Development Module - in Python/Week4/Shop.py
"""
Shop.py
Billy Cussen
27/09/2020
"""

#Class Declaration
class myShop:

    #Constructor
    def __init__(self, hoodies, tshirts, caps):
        self.hoodies = hoodies
        self.tshirts = tshirts
        self.caps = caps
        self.PRICEPERHOODY = 25.99
        self.PRICEPERTSHIRT = 12.99
        self.PRICEPERCAP = 6.99

    #Compute
    def computeTotal(self):
        return (self.hoodies*self.PRICEPERHOODY)+(self.tshirts*self.PRICEPERTSHIRT)+(self.caps*self.PRICEPERCAP)

File: Java/Software Development Module - Java/Week4/Shop.java
/*
Shop.java
Billy Cussen
10/07/2020
*/

public class Shop {
    
    //Data Members
    private int hoodies;
    private int tshirts;
    private int caps;
    private final double COSTPERHOODIES;
    private final double COSTPERTSHIRT;
    private final double COSTPERCAP;
    private double total;

    //Constructor
    public Shop(int hoodies, int tshirts, int caps){
        this.hoodies=hoodies;
        this.tshirts=tshirts;
        this.caps=caps;
        this.COSTPERHOODIES = 25.99;
        this.COSTPERTSHIRT = 12.99;
        this.COSTPERCAP = 6.99;
        this.total = 0;
    }

    //Compute
    public void computeTotal(){
        this.total = (this.hoodies * this.COSTPERHOODIES)+(this.tshirts * this.COSTPERTSHIRT)+(this.caps * this.COSTPERCAP);
    }

    //Getter
    public double getTotal(){
        return this.total;
    }

}